src/target/riscv/riscv-013.c

```diff
@@ -57,6 +57,12 @@ static void riscv013_fill_dmi_write_u64(struct target *target, char *buf, int a,
 static void riscv013_fill_dmi_read_u64(struct target *target, char *buf, int a);
 static int riscv013_dmi_write_u64_bits(struct target *target);
 static void riscv013_fill_dmi_nop_u64(struct target *target, char *buf);
+static int riscv013_test_sba_config_reg(struct target *target, target_addr_t legal_address,
```

Comment:
I would move these down, as these are specific to the test command and seem weirdly mixed in with the functional stuff